docs/getting_started/install_with_ddev.md

```diff
@@ -0,0 +1,405 @@
+See [DDEV database types documentation](https://ddev.readthedocs.io/en/latest/users/extend/database-types/) for available version ranges.
+
+#### Configure database connection
+
+Now, configure the database connection for your Ibexa DXP project. Depending on your database of choice (MySQL or PostgreSQL), use the appropriate command below.
+
+!!! note
+
+    Those commands set a `DATABASE_URL` environment variable inside the container which overrides [the variable from `.env`](install_ibexa_dxp.md#change-installation-parameters).
+
+    To use `.env.local` file instead of server-level environment variables, see [Using dotenv](#using-dotenv).
+
+=== "MariaDB / MySQL"
+
+    ```bash
+    ddev config --web-environment-add DATABASE_URL=mysql://db:db@db:3306/db
+    ```
+
+=== "PostgreSQL"
+
+    ```bash
+    ddev config --web-environment-add DATABASE_URL=postgresql://db:db@db:5432/db
+    ```
+
+#### Enable Mutagen (optional)
+
+If you're using macOS or Windows, you might want to enable [Mutagen](https://ddev.readthedocs.io/en/latest/users/install/performance/#mutagen) to improve performance. You can do this by running the following command:
+
+```bash
+ddev config --mutagen-enabled
+```
+
+See [DDEV performance documentation](https://ddev.readthedocs.io/en/latest/users/install/performance/) for more.
+
+#### Change port mapping (optional)
+
+By default, DDEV uses ports 80 and 443.
+You can [set different ports](https://ddev.readthedocs.io/en/latest/users/usage/troubleshooting/#method-2-fix-port-conflicts-by-configuring-your-project-to-use-different-ports) with a command like the following:
+
+```bash
+ddev config --http-port=8080 --https-port=8443
+```
+
+### 3. Start DDEV
+
+Proceed by starting your DDEV environment with the following command:
+
+```bash
+ddev start
+```
+
+!!! tip
+
+    If you forgot some part of configuration, you can set it by using `ddev config` later, but afterwards you have to restart DDEV with `ddev restart`.
+
+### 4. Composer authentication
+
+Next, you need to [set up authentication tokens](install_ibexa_dxp.md#set-up-authentication-tokens) by modifying the Composer configuration.
+You must run the following command **after** executing `ddev start`, because the command runs inside the container.
+Replace `<installation-key>` and `<token-password>` with your actual installation key and token password, respectively.
+
+```bash
+ddev composer config --global http-basic.updates.ibexa.co <installation-key> <token-password>
+```
+
+This authentication won't persist if the project is restarted (by `ddev restart` or `ddev composer create`). You can back up the authentication file (`auth.json`) by using the following command:
+
+```bash
+ddev exec "mkdir -p .ddev/homeadditions/.composer; cp ~/.composer/auth.json .ddev/homeadditions/.composer"
+```
+
+If you want to reuse an existing `auth.json` file, see [Using an auth.json](#using-an-authjson).
+
+### 5. Create Ibexa DXP project
```

Comment:
```suggestion
### 5. Create project
```

If possible, try to avoid using the product name in headers and titles to limit the amount of entries that will have to be added to plugins.yml if we ever decide to change the product name.